src/main/resources/data/tconstruct/scripts/tconstruct/melting.zs

```diff
@@ -0,0 +1,32 @@
+/*
+ * Adds four Melting recipes that do the following:
+ * 
+ * 1) Adds a normal Melting Recipe that produces 1mb of Water (Fluid) when a Diamond is melted at 0°C.
+ * 2) Adds a Damageable Melting Recipe that producess 1000mb of Water (Fluid) when a Diamond Axe is melted at 0°C.
+ * 2.1) Damageable Melting Recipes are recipes who's output amount (the amount of Fluid that is produced) changes depending on how damaged the input item is.
+ * 3) Adds an Ore Melting recipe that produces 1mb of Water (Fluid) when a Stick is melted at 0°C.
+ * 3.1) Ore Melting Recipes are recipes who's output amount (the amount of Fluid that is produced) changes depending on which Controller (Smeltery or Melter) the recipe is made in.
+ * 3.2) This recipe produces a byproduct of 50mb of Lava (Fluid).
+ * 4) Adds a Material Melting Recipe that allows an Axe Head to be melted into 5 ingots worth of the material the Axe Head was made of.
+ * 4.1) Material Melting Recipes are recipes who's output Fluid depends on the Material of the input Item, for example, if you smelt a Gold Axe Head, the recipe will produce 5 ingots worth (720mb) Molten Gold (Fluid).
+ * 4.2) This script is commented out as it conflicts with default Tinkers Recipes.
```

Comment:
Is it possible to add and remove the recipe so you can make the axe head melt for more? Otherwise this handler will get little use honestly as all addons with new tool parts should be adding a copy for their parts